api/fixtures/example.go

```diff
@@ -71,6 +71,7 @@ type ExampleOptions struct {
 	AutoRepair                       bool
```

Comment:
Please update commit messages to follow imperative mood https://github.com/openshift/hypershift/blob/main/docs/content/contribute/index.md e.g Add arm64 support for Azure NodePool

Reply:
This has been updated.